app/src/main/java/edu/ucandroid/weathernotice/ui/main/MainFragment.kt

```diff
@@ -28,7 +28,7 @@ class MainFragment : Fragment() {
     }
 
     lateinit var mFusedLocationProviderClient: FusedLocationProviderClient
-    val locationRequestId = 100
+    private val locationRequestId = 100
 
```

Comment:
Changed the access modifier to private so that it is only visible within this class. 

Citation: [https://kotlinlang.org/docs/visibility-modifiers.html#packages](https://kotlinlang.org/docs/visibility-modifiers.html#packages)

Reply:
ok, I didn't notice this at first when  trying make it work the first time and ended up forgetting about this, good catch.

Reply:
It can definitely easily be overlooked, that's what the review is for!